gettsim/social_insurance_contributions/eink_grenzen.py

```diff
@@ -149,39 +211,172 @@ def midi_job_bemessungsentgelt_m(
 
     Returns
     -------
-    Income subject to social insurance contributions for midi job.
+    Income subject to social insurance contributions for midijob.
```

Comment:
Only from October 2022 on?

Reply:
Couldn't we just still call this `midijob_bemessungsentgelt_m` and add `10_2022` of course?